src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/ProcessRdfForm.java

```diff
@@ -46,53 +46,53 @@
 import edu.rpi.twc.dcods.vivo.ServerInfo;
```

Comment:
Looks like changes to this file are just removing spaces at the end of lines. What changed specifically for project update form?